include/laser_filters/scan_shadows_filter.h

```diff
@@ -82,26 +81,44 @@ class ScanShadowsFilter : public filters::FilterBase<sensor_msgs::LaserScan>
       ROS_ERROR("Error: ShadowsFilter was not given window.\n");
       return false;
     }
-    neighbors_ = 0;//default value
+    neighbors_ = 0;  // default value
     if (!filters::FilterBase<sensor_msgs::LaserScan>::getParam(std::string("neighbors"), neighbors_))
     {
       ROS_INFO("Error: ShadowsFilter was not given neighbors.\n");
     }
 
+    if (min_angle_ < 0)
```

Comment:
The PR adds a restriction of min and max angle to the shadow detection threshold.

`min_angle` larger than 90 degrees seems odd for the shadow detection; however, this might break unintended usages.
Is it better to add a parameter to switch old algorithm and the new one?